src/webcomponents/clinical/clinical-analysis-create.js

```diff
@@ -592,24 +593,41 @@ export default class ClinicalAnalysisCreate extends LitElement {
                                 defaultValue: "No proband or sample selected.",
                                 columns: [
                                     {
-                                        id: "id",
                                         title: "ID",
-                                        field: "id",
-                                        formatter: (value, row) => `<span style="font-weight: bold">${row.id}</span>`,
+                                        type: "complex",
+                                        display: {
+                                            defaultValue: "-",
+                                            template: "${id} ${somatic}",
+                                            format: {
+                                                "somatic": (somatic, sample) => sample.somatic ? "Somatic" : "Germline",
```

Comment:
For type SINGLE the selected sample is always Germline, so there is no need to display the sample type there.